test/tests/api/v2_0/skupack_tests.py

```diff
@@ -0,0 +1,422 @@
+from config.api2_0_config import *
+from modules.logger import Log
+from on_http_api2_0 import ApiApi as Api
+from on_http_api2_0.rest import ApiException
+from proboscis.asserts import *
+from proboscis import SkipTest
+from proboscis import test
+from json import dumps, loads
+from on_http_api2_0 import rest
+import time
+import requests
+import os
+import subprocess
+import json
+import tarfile
+import shutil
+
+
+LOG = Log(__name__)
+
+
+@test(groups=['skus_api2.tests'])
+class SkusTests(object):
+    def __init__(self):
+
+        self.__client = config.api_client
+        self.clear()
+        self.__nodes= []
+        self.__sku_id = ""
+        self.__workflows = {}
+        self.__tasks ={}
+        self.__config_json = {}
+        self.__skuPackTarball = "/tmp/tarball/mytest.tar.gz"
```

Comment:
Whether this means we have to manually put the tarball to the right location, otherwise the test will fail? As for me, this is not a good design to bury this dependency into the test case.
@jlongever , what's your practice for this situation or do you have any better suggestion?